package.json

```diff
@@ -14,6 +14,9 @@
     "plugin",
     "graphiql"
   ],
+  "files": [
```

Comment:
Isn't required the `umd` folder?

the instructions are not very clear about it.

> Use a build script to generate your UMD build in the umd directory when you publish

but before it says 
> Add the umd (or dist) directory to your .gitignore file

Reply:
Maybe it just exposes everything

Reply:
I think it works for dist or umd (dist is more popular so I'd try with dist), dist is in the gitignore